cv_bridge/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ else()
 endif()
 
 find_package(sensor_msgs REQUIRED)
-find_package(OpenCV 3 REQUIRED
+find_package(OpenCV 4 REQUIRED
```

Comment:
This line force to OpenCV4 , there is no option for version 3.

You should do something like:

```
find_package(OpenCV 4 QUIET opencv_core opencv_core)
if(NOT OpenCV_FOUND)
  find_package(OpenCV 3 REQUIRED opencv_core opencv_core)
endif()
```

Reply:
It really works, I checked it and uploaded one new patch.
@ahcorde